cli/src/pcluster/templates/slurm_builder.py

```diff
@@ -95,6 +96,22 @@ def _stack_unique_id(self):
     def _format_arn(self, **kwargs):
         return Stack.of(self).format_arn(**kwargs)
 
+    # -- Parameters -------------------------------------------------------------------------------------------------- #
+
+    def _add_parameters(self):
+        CfnParameter(
+            scope=self.stack_scope,
+            id="ConfigVersion",
+            description="Version of the original config used to generate the stack",
+            default=self.config.original_config_version,
+        )
+        self.cluster_dns_domain = CfnParameter(
+            scope=self.stack_scope,
+            id="ClusterDNSDomain",
```

Comment:
I don't think this should be a parameter. It seems to me more an output than an input parameter and we don't need it for failed stacks.